modules/apps/export-import/export-import-api/src/main/java/com/liferay/exportimport/configuration/ExportImportServiceConfiguration.java

```diff
@@ -79,5 +79,15 @@ public interface ExportImportServiceConfiguration {
 		name = "validate-missing-references", required = false
 	)
 	public boolean validateMissingReferences();
+	
```

Comment:
It could be helpful to put some comments to describe this configure, as long as it can help others when looking into the impelementation. Sample here https://github.com/liferay/liferay-portal/blob/master/modules/apps/document-library/document-library-api/src/main/java/com/liferay/document/library/configuration/DLConfiguration.java#L86-L90